lib/samson/tasks/lock_cleaner.rb

```
@@ -0,0 +1,26 @@
+module Samson::Tasks
+  class LockCleaner
+
+    def self.start
+      new.start
+    end
+
+    def start
+      task.tap(&:execute)
+    end
+
+    def task
+      @task ||= Concurrent::TimerTask.new(run_now: true, execution_interval: 60, timeout_interval: 10) do
+        Lock.remove_expired_locks
+      end.with_observer(self)
```

with_observer will be called after the first execution which might hide errors